nibabel/freesurfer/tests/test_io.py

```diff
@@ -272,7 +272,7 @@ def test_write_annot_fill_ctab():
         print(labels)
         with clear_and_catch_warnings() as w:
             write_annot(annot_path, labels, rgbal, names, fill_ctab=False)
-        assert_true(
+        assert (
             any('Annotation values in {} will be incorrect'.format(
                 annot_path) == str(ww.message) for ww in w))
```

Comment:
This can become:

```Python
assert any('Annotation values in {} will be incorrect'.format(annot_path) == str(ww.message)
           for ww in w)
```

Similarly below.